config/application.rb

```diff
@@ -25,5 +25,18 @@ class Application < Rails::Application
         path.start_with?(Rails.root.join('spec/javascripts').to_s)
       end
     end
+
+    config.middleware.insert_before 0, 'Rack::Cors' do
+      allow do
+        origins '*'
```

Comment:
I also tend to wildcard this, but I am aware of security evaluators that will frown at the wildcard. What I have done in response is dynamically set the origin to whatever the domain is of the incoming request (effectively being a wildcard). FYI only.